src/Courant/Courant_presets.jl

```diff
@@ -0,0 +1,54 @@
+module Courant
+
+
+export Advective_CFL, Diffusive_CFL
+
+using LinearAlgebra, StaticArrays
+using ..VariableTemplates
+using ..MoistThermodynamics
+using ..PlanetParameters
+import ..MoistThermodynamics: internal_energy
+using ..SubgridScaleParameters
+using GPUifyLoops
+using ..MPIStateArrays: MPIStateArray
+using CLIMA.Atmos: AtmosModel, soundspeed
+import CLIMA.DGmethods: BalanceLaw, vars_aux, vars_state,
+                        vars_diffusive
+using ..Mesh.Topologies
+using ..Mesh.Grids
+using ..Mesh.Grids: VerticalDirection, HorizontalDirection, EveryDirection
+using Logging
+using Printf
+using LinearAlgebra
+using CLIMA.Atmos
+
+
+
+function Advective_CFL(m::AtmosModel, state::Vars, aux::Vars,
```

Comment:
Note that both of these definitions of Advection CLF and Diffusive CFL are atmos specific functions. Might be better to have them live in the atmos model and create similar functions for the ocean?

Style-wise `Advective_CFL` and `Diffusive_CFL` should be lower case since they are not types.

Reply:
@jkozdon I see what you mean, I'll make the changes as requested

Reply:
Better yet: make the functions universal here, i.e., calculate the relevant soundspeed and viscosity/diffusivity in the atmosphere/ocean models, then call the functions here with those model-specific quantities. 

Please also change `_CFL` to `Courant_number` for consistency with the rest. 

BTW: we want to share the turbulence module between atmosphere and ocean in the end. The functions we need there are the same for atmosphere and ocean; just parameters are different.

Reply:
@tapios @jkozdon I could probably change the functions to use any BalanceLaw instead of AtmosModel allowing them to be used by any model, would this be alright?

Reply:
Yes, that sounds good to me.